### PR TITLE
feat: style changes to ds preview page

### LINKED ISSUE #459: Preview Page Layout & Style Tweaks
Splits up version info and metadata into two boxes:

image

 
If there is no readme, the version info and metadata boxes will be full width:
image

- Other style tweaks necessary to more closely match the mockups.

### PR DESCRIPTION
**Pull request by boandriy**
style changes to ds preview page

closes #459

### CODE REVIEW

**Review by chriswhong — APPROVED** [review]
LGTM

File: src/chrome/ContentBoxSubTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+interface ContentBoxSubTitleProps {
+  title: string
+}
+
+const ContentBoxSubTitle: React.FC<ContentBoxSubTitleProps> = ({
+                                                           title
+                                                         }) => (
```
Comment (b5) [inline src/chrome/ContentBoxSubTitle.tsx]: strange spacing here

File: src/chrome/DatasetCommitInfo.tsx [hunk 2]
```diff
@@ -14,21 +14,46 @@ interface DatasetCommitInfoProps {
   // isRow indicates whether this is being displayed inline in a row in a list (versus in a card)
   // and is used to tweak styles
   inRow?: boolean
+  // custom styles are applied if the commit info is for preview page
+  preview?:boolean
+  flex?:boolean
```
Comment (b5) [inline src/chrome/DatasetCommitInfo.tsx]: need space: `: boolean`

**Review by b5 — APPROVED** [review]
a few small nits, feel free to merge once fixed

**Pull request merged**